domains/measurements/Measurements.API/Tests/MeteringPoints/Api/v2024_01_10/MeteringPointControllerTests.cs

```diff
@@ -0,0 +1,201 @@
+using System;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using API.MeteringPoints.Api.v2024_01_10.Dto.Responses;
+using FluentAssertions;
+using NSubstitute;
+using VerifyTests;
+using VerifyXunit;
+using Xunit;
+using Tests.Extensions;
+using API;
+using Microsoft.Extensions.DependencyInjection;
+using Tests.Fixtures;
+using System.Linq;
+using System.Net;
+
+namespace Tests.MeteringPoints.Api.v2024_01_10;
+
+[UsesVerify]
+public class MeteringPointControllerTests : MeasurementsTestBase
+{
+    public MeteringPointControllerTests(TestServerFixture<Startup> serverFixture)
```

Comment:
instead of a TestServerFixture, cant we use the build-in IClassFixture?

Reply:
No. If we can I'm not sure how